File: HelloWars/SampleWebBotClient/Models/TankBlaster/Bomb.cs
using System.Drawing;

namespace SampleWebBotClient.Models.TankBlaster
{
    public class Bomb
    {
        public Point Location { get; set; }
        public int RoundsUntilExplodes { get; set; }
        public int ExplosionRadius { get; set; }
    }
}

File: JAVA/TankBlasterBot/src/bot/Bomb.java
package bot;

import java.awt.Point;

public class Bomb {
	public String Location;
	public int RoundsUntilExplodes;
	public int ExplosionRadius;

	public Point LocationToPoint() {
		String[] coords = Location.split(",");
		return new Point(Integer.parseInt(coords[0].trim()),
				Integer.parseInt(coords[1].trim()));
	}
}